README-es.md

```diff
@@ -0,0 +1,33 @@
+# go-perfbook
+
+[![Cómprame un café](https://www.buymeacoffee.com/assets/img/custom_images/orange_img.png)](https://www.buymeacoffee.com/dgryski)
+
+Este documento describe las mejores prácticas para escribir código de alto rendimiento en Go.
+
+Las primeras secciones cubren cómo escribir código optimizado en cualquier lenguaje.
+Las secciones posteriores cubren las técnicas específicas de Go.
+
+### Versiones en varios idiomas
+
+* [English](README.md)
+* [中文](performance-zh.md)
+* [Español](README-es.md)
+
+### Tabla de contenidos
+
+1. [Escribir y optimizar código en Go](performance-es.md#escribir-y-optimizar-codigo-en-go)
+
+### Cómo contribuir
+
+Este es un libro en desarrollo sobre rendimiento y optimización en Go.
```

Comment:
Este es un libro en desarrollo de software especializado en rendimiento y optimización en Go

Reply:
I think the word `desarrollo` actually refers to the state of the document. The document is under development.

Reply:
As I understood in the original text, the work-in-progress refers to the book itself. I find the suggestion changes the original meaning in this case. WDYT?

Reply:
🤔 I think the current version is clear enough about the book been under development.